src/mongoose_client_api/mongoose_client_api.erl

```diff
@@ -77,6 +78,13 @@ set_resp_body_if_missing(Body, Req) ->
             cowboy_req:set_resp_body(Body, Req)
     end.
 
+maybe_report_error(StatusCode, Req, Body) when StatusCode >= 400 ->
+    ?LOG_WARNING(#{what => reply_error,
+                   stacktrace => element(2, erlang:process_info(self(), current_stacktrace)),
+                   code => StatusCode, req => Req, reply_body => Body});
```

Comment:
This one is logging the `Body` with `reply_body` as a field, should be `body`, right?

Reply:
nope, there is no body field documented yet.
So, reply_body. Because there could be also request_body.

Reply:
```erlang
           {remove_fields_filter, {fun mongoose_log_filter:remove_fields_filter/2, [req, body]}},
```
From the config, I understand that `req` and `body` are the only ones removed.

Reply:
fixed